controllers/comments-controller.js

```diff
@@ -29,3 +29,17 @@ exports.postComment = (req, res, next) => {
       });
 };
 
+exports.deleteComment = (req, res, next) => {
+  const { comment_id } = req.params;
+  const promises = [removeComment(comment_id), checkCommentExists(comment_id)];
+  Promise.all(promises)
+    .then(() => {
+
+      res.status(204).send("");
+    })
+    .catch((err) => {
+      next(err);
+    });
```

Comment:
I can see your controller and model for delete comments here. Please make sure that only the code for the specified ticket is included in each pull request. Has your code for ticket 9 (delete comments) been reviewed already?